internals/src/slice.rs

```diff
@@ -58,6 +77,27 @@ impl<T> SliceExt for [T] {
         let left = unsafe { core::slice::from_raw_parts_mut(left.as_mut_ptr().cast::<[Self::Item; N]>(), chunks_count) };
         (left, right)
     }
+
+    fn get_array<const ARRAY_LEN: usize>(&self, offset: usize) -> Option<&[Self::Item; ARRAY_LEN]> {
```

Comment:
> One is in iterator - trying to go past the end means end of iteration. The other is get method which is similar enough to get on slices to warrant matching API. std has Option there not Result. While returning indices in error types is somewhat convenient in erroneous cases using get is usually an operation that's handled by the caller.

I see, you're talking about when methods impl the trait.  I guess doc comments here aren't need since it's implied by the trait.

Reply:
I was talking about where these methods are used.